packages/appstore/src/components/total-assets/total-assets.tsx

```diff
@@ -156,7 +153,7 @@ const TotalAssets = ({ category }: TTotalAssets) => {
                     icon='info'
                     disable_message_icon
                     is_bubble_hover_enabled
-                    message={is_eu_popover_text}
+                    message={popover_text}
```

Comment:
Ditto, same here. It is more readable this way since you do not need to scroll to know what the message is